site/content/api/Event-API.md

```diff
@@ -89,42 +89,28 @@ See [*Telemetry API*]({{< relref "Telemetry-API.md" >}}) for definition of messa
 
 Hono defines *well-known* events that are of a specific *content-type*.
 
-NB: currently there is only one such *well-known* event.
+In the following these events are specified in detail.
 
-### Lifecycle Notifications
+### Device connection notification
 
-A specific event can signal that a device is `connected` (e.g. to signal that it is ready to receive a command for a specific time interval) 
-or that a device is `disconnected` (e.g to signal that it is not ready to receive a command anymore).
- 
-*Business Applications* may want to react to such an event by sending a command upstream to the device.
-For that purpose, the protocol adapter instance that received such an event opens a receiver link for the device (scoped to its tenant) that
-is used to send the command to the appropriate protocol adapter instance.
+By sending an event of this type the receiver is notified that a device is `connected` or `disconnected` to Hono.
 
-Such an event may be issued by the device itself or by a protocol adapter detecting that a device is connected or disconnected.
- For further details on which part of a setup issues such an event see (... concept page).
+Such an event is typically issued by a protocol adapter.
 
+***Device connection notification payload***
 
-*Lifecycle notifications* may be decided to not being durable, but this is left to the setup of the *AMQP network* and is not further specified in the following.
-
-
-***Lifecycle Notification Payload***
-
-The following table provides an overview of the properties a client needs to set for a *lifecycle notification* event.
+The following table provides an overview of the properties a client needs to set for a *device connection notification* event.
```

Comment:
If later on the source explicitly is a "protocol adapter", then I think here "client" should also become "protocol adapter".

Reply:
Currently this would be true, but what if we have other "clients" later on, that are not protocol adapters, sending this event?

How about changing it to `event sender`, so resulting in 
`... overview of the properties an event sender needs to set ...`?
Then we would be generic here, and that currently the protocol adapters are the only `event senders` would be made clear in the concept page (which I provide in another commit, needs some rework still).
WDYT?